emeritus.md

```diff
@@ -6,3 +6,4 @@ Emeritus committee members:
 - Jordan Liggitt (**[@liggitt](https://github.com/liggitt)**) `<jordan@liggitt.net>` [4096R/0x39928704103C7229]
 - Swamy Shivaganga Nagaraju (**[@swamymsft](https://github.com/swamymsft)**) `<gaswamy@microsoft.com>`
 - Tim Allclair (**[@tallclair](https://github.com/tallclair)**) `<timallclair@gmail.com>`
+- Luke Hinds (**[@lukehinds](https://github.com/lukehinds)**) `lhinds@protonmail.com`
```

Comment:
alphabetical based on first name?

Reply:
ah that makes sense - I thought this was just in order of departure. 😅

Reply:
Looking back it does seem like departure order was used most recently (https://github.com/kubernetes/committee-security-response/pull/173) but I don't feel strongly either way - should we switch it?

Reply:
oh interesting. I also dont care either way as long as we are consistent. Maybe add a comment somewhere so we remember the order used for this?

Reply:
I think this is fine as-is.